Modules/xASL_module_BIDS2Legacy.m

```diff
@@ -83,18 +80,20 @@
         % x.modules.bids2legacy.BIDS.subjects(1).perf(1) -> filename: 'sub-10015124_ses-1_run-1_asl.nii.gz'
         % x.modules.bids2legacy.BIDS.subjects(1).perf(2) -> filename: 'sub-10015124_ses-1_run-2_asl.nii.gz'
         % x.modules.bids2legacy.BIDS.subjects(1).perf(3) -> filename: 'sub-10015124_ses-1_run-1_m0scan.nii.gz'
-        % x.modules.bids2legacy.BIDS.subjects(1).perf(4) -> filename: 'sub-10015124_ses-1_run-2_m0scan.nii.gz'            
+        % x.modules.bids2legacy.BIDS.subjects(1).perf(4) -> filename: 'sub-10015124_ses-1_run-2_m0scan.nii.gz'  
+		%
+		% Finally note that visits can now be any name, number, data, or a string and are always treated as a string
```

Comment:
Remove the word "Finally" as this is grammatically weird

-> is this a final note in a list of notes? Or should the user always wait with noting this until the "final" moment ;)

Or did you mean "Note that visits at this point can be any ..."

-> which is a moot statement. This code is completely independent of dicom2BIDS so this is the beginning, not final of something?

Reply:
Replaced by "also"